recipes-bsp/uefi/tegra-uefi-capsules/oe4t-set-uefi-OSIndications

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+printf "\x07\x00\x00\x00\x04\x00\x00\x00\x00\x00\x00\x00" > /tmp/var_tmp.bin
+dd if=/tmp/var_tmp.bin of=/sys/firmware/efi/efivars/OsIndications-8be4df61-93ca-11d2-aa0d-00e098032b8c bs=12;sync
```

Comment:
The destination path has to be different for the AGX Xavier, which doesn't support writing variables in the efivars sysfs.

It might be better to move this script to the `setup-nv-boot-control` recipe and have it leverage the logic in the existing script there (maybe with a bit of refactoring) to determine how to modify the EFI variable.

Reply:
See what you think of the refactoring.  I initially wanted to make a `.func` file for both compat spec logic and for setting the uefi var, but they were intertwined just enough that I left it together.

Reply:
At first glance it looks good - just the sort of thing I had in mind..  We should probably use the full path to the `.func` file in the `source` line, though.  I'll review in more detail when I get a chance.

Reply:
Full path has been taken care of.  Not sure why, but the auto-builder / checklayer broke after I forced push.  The commits all look correct so not sure what happened.  I am still chasing down an issue during my testing.  Invoking the script to set `OsIndications` works the first time through, but not the second.  I am tracking that down now.

Reply:
OK... I should have looked at the existing logic a little closer (and the comment).  I'll have to rework that logic as it looks like the design intent was to only write `TegraPlatformSpec` and `TegraPlatformCompatSpec` the first time -  presumably because those should be static values.

Reply:
> Not sure why, but the auto-builder / checklayer broke after I forced push. The commits all look correct so not sure what happened.

Force-pushes (and some other unusual events) can confuse the autobuilder's PR handler.  Fixing that is on my to-do list, but in the meantime, I re-queued the job.

Reply:
Logic should be fixed now.  Tested A->B and B->A on Orin AGX Devkit.  Will test Xavier AGX Devkit next.

Reply:
Tested A->B and B->A on Xavier AGX Devkit as well and everything looks OK.  I think this is gtg pending any additional feedback.